app/models/user.rb

```diff
@@ -86,6 +86,7 @@ class User < ApplicationRecord
   after_create :create_profile!
 
   scope :members, -> { where(role: %i[member admin]) }
+  scope :super_admins, -> { where(email: ['daniel@agencyoflearning.com', 'dave@agencyoflearning.com']) }
```

Comment:
I added this scope to be able to send out mailer alert to Daniel and Dave in production. I'll add them to the seed file so that they're available for local test too